exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/LogAdapter.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.internal;
```

Comment:
I just noticed (very sorry for seeing this so late) that this got put into the otlp-common module. I don't think this class belongs in common does it? Shouldn't this be in a new otlp-logs module, in parallel to otlp-traces and otlp-metrics?

Reply:
@ran8080 I don't think we can merge this PR in its current state due to the introduction of additional transitive dependencies added to the otlp-common module. Will you have time to correct this issue and put this code into a new module, specific to logging?